src/containers/Network/UpgradeStatus.tsx

```diff
@@ -111,32 +111,28 @@ export const UpgradeStatus = () => {
   return (
     <div className="network-page">
       <Streams validators={vList} updateValidators={updateValidators} />
-      {validatorCount && (
+      {
         // @ts-ignore - Work around for complex type assignment issues
         <Hexagons data={validations} list={vList} />
-      )}
-
+      }
       <div className="stat">
-        {validatorCount && (
-          <>
-            <span>{t('validators_found')}: </span>
-            <span>
-              {localizeNumber(validatorCount, language)}
-              {unlCount !== 0 && (
-                <i>
-                  {' '}
-                  ({t('unl')}: {unlCount})
-                </i>
-              )}
-            </span>
-          </>
-        )}
+        <>
+          <span>{t('validators_found')}: </span>
+          <span>
+            {localizeNumber(validatorCount, language)}
+            {unlCount !== 0 && (
+              <i>
+                {' '}
+                ({t('unl')}: {unlCount})
+              </i>
+            )}
+          </span>
+        </>
       </div>
-
       <div className="wrap">
         <NetworkTabs selected="upgrade_status" />
         <div className="upgrade_status">
-          {vList && (
+          {validatorCount && stableVersion && (
```

Comment:
There should be a "Loading" component if the network is still loading. That's a different state than "we couldn't fetch the data for some reason" and should be reflected accordingly.

Reply:
Thanks Mayukha, update the code and also the PR description.